client/client.go

```diff
@@ -211,6 +211,21 @@ func (e *Error) Error() string {
 	return e.Message
 }
 
+// these should be self-explanatory; otherwise, let us know!
```

Comment:
That comment should be implied over every non-documented interface. :-)  Can we please drop it?